lib/ci/container/ephemeral.rb

```diff
@@ -17,10 +17,13 @@ class << self
 
     def stop(options = {})
       super(options)
-      kill!(options)
+      kill!(options) if running?
       rescued_remove
     end
 
+    def running?
+      json['State']['Running']
```

Comment:
well, you probably should raise if it paused to make it obvious what the problem is